src/event.ts

```diff
@@ -12,8 +12,14 @@ export interface H3Event {
 
 export type CompatibilityEvent = H3Event | IncomingMessage
 
-type _JSONValue<T=string|number|boolean> = T | T[] | Record<string, T>
-export type JSONValue = _JSONValue<_JSONValue>
+interface JSONObject { [x: string]: JSONValue }
+interface JSONArray extends Array<JSONValue> { }
+export type JSONValue =
+  | string
```

Comment:
Style: would you please make it one line too?

Reply:
Done